public/index-pl.html

```diff
@@ -0,0 +1,104 @@
+
+<h2>Potrzebujesz pomocy?</h2>
+
+<p>Nasze dane kontaktowe znajdują się na stronie internetowej spotkań lub w wysłanych wcześniej mailach.</p>
+
+<h2>Wersja krótsza</h2>
+
+<p>Nasza społeczność pragnie tworzyć dostępną dla wszystkich przestrzeń i społeczność, wolną od nękania, niezależnie od płci, wieku, orientacji seksualnej, niepełnosprawności, aparycji, wielkości ciała, rasy, wierzeń (bądź ich braku) oraz wyborów technologicznych. Nie tolerujemy molestowania uczestników konferencji w żadnej formie. Seksistowski język jest nieodpowiedni niezależne od miejsca: spotkań, prezentacji, warsztatów, imprezy bądź jakiegokolwiek miejsca związanego ze społecznością (także w Internecie). Osoby nieprzestrzegające tych zasad będą (za zgodą organizatorów) ukarani lub usunięci ze społeczności, spotkania oraz ze wszystkich przyszłych spotkań, <em>bez zwrotu kosztów</em>. 
+
+<h2>Wersja dłuższa</h2>
+
+<p>Nękanie obejmuje obraźliwe komentarze nawiązujące do płci, orientacji seksualnej, wieku, niepełnosprawności, aparycji, wielkości ciała, rasy, religii, wyborów technologicznych, obsceniczne obrazy w miejscach publicznych, celowe zastraszanie, prześladowanie, śledzenie, nękające zdjęcia lub nagrania, ciągłe zakłócanie wykładów lub innych wydarzeń, niestosowny kontakt fizyczny oraz molestowanie seksualne.</p>
+
+<p>Osoby uczestniczące w społeczności proszone o zaprzestanie nieodpowiedniego zachowania, zobowiązane są do natychmiastowego podporządkowania się zasadom.</p>
+
+<p>Zasady odnoszące się do nękania obowiązują wszystkich, włączając w to sponsorów. Sponsorzy w szczególności nie powinni wykorzystywać zdjęć, promować działań oraz wykorzystywać jakichkolwiek materiałów o seksualnych podtekstach. Organizatorzy, uczestnicy, sponsorzy oraz obsługa (włączając w to wolontariuszki i wolontariuszy) nie powinna wyglądać wyzywająco (co odnosi się przede wszystkim do ubrań, strojów lub kostiumów) ani w jakikolwiek inny sposób tworzyć środowiska o seksualnym nacechowaniu.</p>
+
+<p>W sytuacji, gdy osoba uczestnicząca w społeczności zachowuje się w sposób nieodpowiedni, opisany powyżej, organizatorzy mają prawo podjąć działania, które uznają za stosowne, włączając w to ostrzeżenie lub wydalenie nękającej osoby ze społeczności, spotkania oraz ze wszystkich przyszłych spotkań, bez zwrotu kosztów.</p>
+
+<p>Jeżeli nękanie dotyczy Ciebie lub zauważasz przejawy takiego zachowania w stosunku do innych osób, lub masz jakiekolwiek inne obawy, skontaktuj się bezzwłocznie z organizatorami. Kontakt do organizatorów znajduje się na stronie internetowej spotkań lub w wysłanych wcześniej mailach.</p>
+
+<p>Organizatorzy chętnie pomogą uczestniczkom i uczestnikom skontaktować się z ochroną lub służbami porządkowymi, zapewnić eskortę lub w jakikolwiek innym sposób towarzyszyć osobom nękanym osobom, aby czuły się bezpiecznie podczas trwania spotkania. Cenimy Twoje uczestnictwo.</p>
```

Comment:
Similar comment here:
```Organizatorzy chętnie pomogą osobom uczestniczącym skontaktować się```
instead of:
```Organizatorzy chętnie pomogą uczestniczkom i uczestnikom skontaktować się```